templates/login.html

```diff
@@ -86,12 +86,16 @@ <h1 class="lead">
   
   </div>
 
-  <div class="footer col-md-12">
+    <div class="footer col-md-12">
     <p>
       This service is run transparently from
-      <a href="https://github.com/2i2c-org/pilot-hubs">
-        github.com/2i2c-org/pilot-hubs
+      <a href="https://github.com/2i2c-org/infrastructure">
+        github.com/2i2c-org/infrastructure
+      </a> by
+      <a href="https://2i2c.org">
+        2i2c.org
```

Comment:
Extended sentence to acknowledge 2i2c as the operator of the ghg.center hub infrastructure.